stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -91,17 +90,25 @@ private Stetho() {
    * <p />
    * For simple use cases, consider {@link #initializeWithDefaults(Context)}.
    */
+  public static InitializerBuilder newInitializerBuilder(
+      Context context,
+      @Nullable AnalyticsLogger analyticsLogger) {
+    return new InitializerBuilder(context, analyticsLogger);
+  }
+
   public static InitializerBuilder newInitializerBuilder(Context context) {
-    return new InitializerBuilder(context);
+    return newInitializerBuilder(context, null);
   }
 
   /**
    * Start the listening server.  Most of the heavy lifting initialization is deferred until the
    * first socket connection is received, allowing this to be safely used for debug builds on
    * even low-end hardware without noticeably affecting performance.
    */
-  public static void initializeWithDefaults(final Context context) {
-    initialize(new Initializer(context) {
+  public static void initializeWithDefaults(
+      final Context context,
+      @Nullable AnalyticsLogger analyticsLogger) {
```

Comment:
You shouldn't add the option here.  This conflicts with the builder strategy and is a Java antipattern which would cause all options to be entered into a giant matrix of overload possibilities (if you're curious I can show you many, many examples of this problem *grin*).  Also, this doesn't even semantically make sense as it is no longer initializing with defaults if one option is provided as a non default.

Instead, look at the InitializerBuilder itself and maybe also the webkit and dumpapp specific builders (both of which seem more appropriate as the analytics API could be customized based on which facility is being used.